### DISCUSSION TITLE
Firmware cannot be downloaded

### DISCUSSION THREAD

**Discussion by lond131421** [fhdm-dev/scoppy#155] [Q&A]
https://oscilloscope.fhdm.xyz/wiki/firmware-versions  [scoppy-picow-v18.uf2](https://github.com/fhdm-dev/scpdl1/raw/master/a/v18/scoppy-picow-v18.uf2)  Firmware cannot be downloaded！！！

**Comment by fhdm-dev** [comment]
Hmmm. I just tested it and it seems OK. Maybe a temporary glitch....or can you still not download it?

**Reply by lond131421** [reply]
I am a Chinese. I can't find the reason why I can't download it. Can I pass 676005391@qq.com Can you send me an email?

**Reply by lond131421** [reply]
I solved the problem, it was a domain name resolution issue, which was resolved by modifying the hosts. Thank you all for your help and response!

**Comment by kd7eir** [comment]
I was also able to download it with no issues. What error did you get?

**Reply by lond131421** [reply]
I solved the problem, it was a domain name resolution issue, which was resolved by modifying the hosts. Thank you all for your help and response!

**Comment by lond131421** [comment]
I am a Chinese. I can't find the reason why I can't download it. Can I pass 676005391@qq.com Can you send me an email?

**Reply by lond131421** [reply]
I solved the problem, it was a domain name resolution issue, which was resolved by modifying the hosts. Thank you all for your help and response!

**Comment by lond131421** [comment]
I solved the problem, it was a domain name resolution issue, which was resolved by modifying the hosts. Thank you all for your help and response!
